filebeat/module/haproxy/log/test/tcplog.log-expected.json

```diff
@@ -0,0 +1,31 @@
+[
+    {
+        "@timestamp": "2018-09-20T15:44:23.285Z", 
+        "fileset.module": "haproxy", 
+        "fileset.name": "log", 
+        "haproxy.backend_name": "app", 
+        "haproxy.backend_queue": 0, 
+        "haproxy.bytes_read": 212, 
+        "haproxy.client_ip": "127.0.0.1", 
+        "haproxy.client_port": 40962, 
+        "haproxy.connection_wait_time_ms": -1, 
+        "haproxy.connections.active": 1, 
+        "haproxy.connections.backend": 0, 
+        "haproxy.connections.frontend": 1, 
+        "haproxy.connections.retries": 0, 
+        "haproxy.connections.server": 0, 
+        "haproxy.default.logsource": "127.0.0.1", 
+        "haproxy.frontend_name": "main", 
+        "haproxy.pid": 25457, 
+        "haproxy.process_name": "haproxy", 
+        "haproxy.server_name": "<NOSRV>", 
```

Comment:
I guess this is that no server name was provided in the config? I guess that for this case we could leave this field empty, or even remove it.
It'd be nice to have also an example of a log line with a proper server name.